.github/workflows/test-suite.yml

```
@@ -53,20 +53,23 @@ jobs:
     - name: Install dependencies
       run: composer install --prefer-dist --no-progress
 
+    - name: Setup environment to use the desired version of NodeJS
+      uses: actions/setup-node@v3
+
+    - name: Installing NPM dependencies
+      run: npm install
```

This will use the lock file and throw an error if it needs to be updated by a PR.

```suggestion
      run: npm ci
```